src/kernel/chainparams.cpp

```diff
@@ -375,6 +375,15 @@ class SigNetParams : public CChainParams {
 
         vFixedSeeds.clear();
 
+        m_assumeutxo_data = {
+            {
+                .height = 160'000,
+                .hash_serialized = AssumeutxoHash{uint256S("0x5225141cb62dee63ab3be95f9b03d60801f264010b1816d4bd00618b2736e7be")},
+                .nChainTx = 1278002,
```

Comment:
In d9e41a0f92081979764aee420a8ccb80898da862 "chainparams: add signet assumeutxo param at height 160_000"

This is `txouts`/`coins_written` (depending on which RPC you used), not the actual `nChainTx`. It should be 2289496.

Reply:
Thanks, fixed.